### PR TITLE
spanish translations

### PR DESCRIPTION
**Pull request by kablekompany**

### CODE REVIEW

File: i18n/es-ES.json [hunk 1]
```diff
@@ -1 +1,104 @@
...
+  "POWERCORD_LINK_NOW": "Vinculalo ahora",
+  "POWERCORD_LINKING_UNLINKED": "Aún no ha vinculado su cuenta.",
+  "POWERCORD_LINKING_WAITING": "Vinculación de su cuenta...",
+  "POWERCORD_LINKING_TIMED_OUT": "Se agotó el tiempo de espera del enlace :( ¿Quizás intente de nuevo?",
+  "POWERCORD_LINKING_ERRORED": "Se produjo un error. Verifique la consola para obtener más detalles y, por favor, [repórtelo] ({newIssueUrl}) si vuelve a ocurrir.",
+  "POWERCORD_COMMAND_PREFIX": "Prefijo de Comando",
+  "POWERCORD_SETTINGS_SYNC": "Sincronización de Configuraciones",
+  "POWERCORD_SETTINGS_SYNC_DESC": "Sincronice todas sus configuraciones de Powercord en todos los dispositivos. Requiere una cuenta Powercord!",
+  "POWERCORD_SETTINGS_NO_CLYDE": "Erradicar Clyde",
+  "POWERCORD_SETTINGS_NO_CLYDE_DESC": "Reemplaza a [Clyde] ({discordiaUrl}) en los comandos de Powercord con una gama mixta de avatares y nombres de usuario seleccionados por los desarrolladores de complementos; retrocede a \"Powercord\" de forma predeterminada.",
```
Comment (cyyynthia) [inline i18n/es-ES.json]: Space should be removed
```suggestion
  "POWERCORD_SETTINGS_NO_CLYDE_DESC": "Reemplaza a [Clyde]({discordiaUrl}) en los comandos de Powercord con una gama mixta de avatares y nombres de usuario seleccionados por los desarrolladores de complementos; retrocede a \"Powercord\" de forma predeterminada.",
```

File: i18n/es-ES.json [hunk 2]
```diff
@@ -1 +1,104 @@
...
+  "POWERCORD_SETTINGS_EXP_WEB_PLATFORM": "Características experimentales de la plataforma web",
+  "POWERCORD_SETTINGS_EXP_WEB_PLATFORM_DESC": "Habilita las características experimentales de la plataforma web de Chromium que están en desarrollo, como CSS \"backdrop-filter\". Como las características están en desarrollo, puede encontrar problemas y las API pueden cambiar en cualquier momento. **Requiere reinicio**.",
+  "POWERCORD_SETTINGS_DISCORD_EXPERIMENTS": "Habilitar experimentos de discordia",
+  "POWERCORD_SETTINGS_DISCORD_EXPERIMENTS_DESC": "****ADVERTENCIA:**** Habilitar esto le da acceso a funciones que Discord puede detectar y que pueden provocar la cancelación de una cuenta****. Powercord **no es responsable** de lo que haces con esta función. Déjelo desactivado si no está seguro. El equipo de Powercord **no brindará asistencia** con respecto a cualquier experimento.",
```
Comment (cyyynthia) [inline i18n/es-ES.json]: Missing some *
```suggestion
  "POWERCORD_SETTINGS_DISCORD_EXPERIMENTS_DESC": "****ADVERTENCIA:**** Habilitar esto le da acceso a funciones que Discord puede detectar y que pueden provocar la ****cancelación de una cuenta****. Powercord **no es responsable** de lo que haces con esta función. Déjelo desactivado si no está seguro. El equipo de Powercord **no brindará asistencia** con respecto a cualquier experimento.",
```

File: i18n/es-ES.json [hunk 3]
```diff
@@ -1 +1,104 @@
...
+  "POWERCORD_SETTINGS_BACKEND": "URL de backend",
+  "POWERCORD_SETTINGS_BACKEND_DESC": "URL utilizada para obtener algunos activos y para consultar la API REST de Powercord.",
+  "POWERCORD_SETTINGS_CACHE_POWERCORD": "Borrar la caché de Powercord",
+  "POWERCORD_SETTINGS_CACHE_POWERCORD_DESC": "Borra el caché de Powercord. El próximo reinicio puede parecer más lento, ya que algunos archivos deberán compilarse nuevamente.",
+  "POWERCORD_SETTINGS_CACHE_DISCORD": "Borrar la caché de Discord",
+  "POWERCORD_SETTINGS_CACHE_DISCORD_DESC": "Elimina todo lo almacenado en la caché de Discord carpeta. Esto hará que Discord se sienta más lento temporalmente, ya que todos los recursos deberán recuperarse de nuevo.",
+  "POWERCORD_SETTINGS_CACHE_CLEARED": "Caché despejado!",
```
Comment (cyyynthia) [inline i18n/es-ES.json]: I'm not an expert at spanish, but shouldn't this begin with an upside down `!`?

File: i18n/es-ES.json [hunk 4]
```diff
@@ -1 +1,104 @@
...
+  "POWERCORD_SETTINGS_RESTART": "Esta configuración requiere que reinicies Discord para que surta efecto. ¿Quieres reiniciar Discord ahora?",
+  "POWERCORD_UPDATES_DISABLED": "Las actualizaciones están deshabilitadas",
+  "POWERCORD_UPDATES_PAUSED": "Las actualizaciones están en pausa",
+  "POWERCORD_UPDATES_PAUSED_RESUME": "Se reanudarán en la próxima recarga",
+  "POWERCORD_UPDATES_CHECKING": "Buscando actualizaciones ... ",
```
Comment (cyyynthia) [inline i18n/es-ES.json]: Space before ...

File: i18n/es-ES.json [hunk 5]
```diff
@@ -1 +1,104 @@
...
+  "POWERCORD_UPDATES_CHECKING_STATUS": "Comprobando {checked}/{total}",
+  "POWERCORD_UPDATES_UPDATING": "Actualizando Powercord ...",
```
Comment (cyyynthia) [inline i18n/es-ES.json]: Space before ...

File: i18n/es-ES.json [hunk 6]
```diff
@@ -1 +1,104 @@
...
+  "POWERCORD_PASSPHRASE": "Frase de contraseña",
+  "POWERCORD_UPDATE_PASSPHRASE": "Actualizar frase de contraseña",
+  "POWERCORD_UPDATE_PASSPHRASE_MODAL1": "Esta frase de contraseña se usará para cifrar sus datos antes de enviarlos a los servidores de Powercord. Se recomienda configurar uno, pero puede dejarlo vacío para enviar sus datos sin cifrar.",
+  "POWERCORD_UPDATE_PASSPHRASE_MODAL2": "Si ya está utilizando la sincronización en otras máquinas, coloque la misma frase de contraseña que usó. **El uso de otra frase de contraseña sobrescribirá los datos antiguos, así que tenga cuidado**.",
+  "POWERCORD_UNLINK": "Desconectar",
+  "POWERCORD_LINK_NOW": "Vinculalo ahora",
+  "POWERCORD_LINKING_UNLINKED": "Aún no ha vinculado su cuenta.",
+  "POWERCORD_LINKING_WAITING": "Vinculación de su cuenta...",
+  "POWERCORD_LINKING_TIMED_OUT": "Se agotó el tiempo de espera del enlace :( ¿Quizás intente de nuevo?",
+  "POWERCORD_LINKING_ERRORED": "Se produjo un error. Verifique la consola para obtener más detalles y, por favor, [repórtelo] ({newIssueUrl}) si vuelve a ocurrir.",
```
Comment (cyyynthia) [inline i18n/es-ES.json]: Space should be removed
```suggestion
  "POWERCORD_LINKING_ERRORED": "Se produjo un error. Verifique la consola para obtener más detalles y, por favor, [repórtelo]({newIssueUrl}) si vuelve a ocurrir.",
```

**Review by cyyynthia — CHANGES_REQUESTED** [review]

**Review by kablekompany — COMMENTED** [review]
Suggested committed, other changes made

**Review by cyyynthia — APPROVED** [review]

**Pull request merged**